docs/user-guide/rules.md

```diff
@@ -11,6 +11,16 @@ We turn on most of the rules in our [standard config](https://www.npmjs.com/pack
 
 You can avoid errors with these `no` rules.
 
+### Deprecated
+
+Disallow deprecated things with these `no-deprecated` rules.
+
+<!-- prettier-ignore-start -->
+| | | |
+| :-- | :-: | :-: |
+| [`declaration-property-value-no-deprecated`](../../lib/rules/declaration-property-value-no-deprecated/README.md)<br/>Disallow deprecated values for properties within declarations. | | 🔧 |
```

Comment:
I don't use standard and I don't have write access to `stylelint-config-standard` so I left it empty because I can't guarantee the release will be synched.